0019-glossary/0019-glossary.md

```diff
@@ -0,0 +1,248 @@
+# Glossary
+
+## Account
+An administrative item that acts as a bucket of assets on a ledger.
+
+## Application Layer
+The set of protocols built upon the [Transport Layer](#transport-layer) that communicate payment details between senders and receivers. Application Layer protocols specify all of the data and methods needed to set up a payment for a specific category of use cases or applications. For example, the Simple Payment Setup Protocol uses WebFinger and HTTPS for account detail lookup.
+
+## Arbitrage
+The practice of doing [cyclic transactions](#cyclic-transaction) which start and end on the same [ledger](ledger),
+or groups of simultaneous transactions which add up to one cyclic transaction.
+Taking advantage of quirks and fluctuations in exchange rates,
+the goal is to achieve a [destination amount](#destination-amount) which is higher than the [source amount](#source-amount).
+
+## Asset
+Something you can own. Can be a physical and unique item, a quantity of a physical substance, a specific physical collection of coins and bank notes,
+a non-physical quantity of currency, a non-physical claim to a resource, a reputation for keeping one's word, someone else's intention to obey the
+terms of a contract, etc. Some types of assets can be more easily traded on a ledger than others. Also, for some asset pairs it is easier to determine
+a fair exchange rate than for others.
+
+## Balance
+The amount of a ledger's assets held by a given account. This concept is not as clearly defined as it may seem at first glance.
+When part of an account's balance is not readily usable, it is up to the ledger implementation to decide whether an account's
+"balance" includes only the readily usable balance, or all of it, or some number inbetween. Some ledgers may give a more detailed
+report, specifying amounts for, for instance, outgoing transfers which have not yet
+completed, incoming transfers which have not yet completed, and for some ledgers even disputed balance.
```

Comment:
This definition seems overly complicated. The additional behavior you describe is not currently implemented anywhere and it's not recommended either